src/Calculator/Calculator.vcxproj.filters

```diff
@@ -1569,5 +1572,9 @@
   <ItemGroup>
     <CopyFileToFolders Include="$(GraphingImplDll)" />
     <CopyFileToFolders Include="$(GraphingEngineDll)" />
+    <CopyFileToFolders Include="$(GraphingImplDll)" />
+    <CopyFileToFolders Include="$(GraphingEngineDll)" />
+    <CopyFileToFolders Include="$(GraphingImplDll)" />
+    <CopyFileToFolders Include="$(GraphingEngineDll)" />
```

Comment:
It seems like Visual Studio duplicates these commands

Reply:
Fixed.

Reply:
Fixed.